recipes/typish/meta.yaml

```diff
@@ -1,42 +0,0 @@
-{% set name = "typish" %}
```

Comment:
Please do not touch files outside of the PRs own recipe

Reply:
I supposed I had to ship just `example` and my own recipe only.

Reply:
No the other recipes are staged.

Reply:
@scopatz Any idea why it's failing on Linux? Do you want me to close this PR?